### DISCUSSION TITLE
How do I generate the token object for testing when using `0x4::token::create` and `0x4::token::create_numbered_token`

### DISCUSSION THREAD

**Discussion by Pelumi527** [aptos-labs/aptos-developer-discussions#341] [Questions]
### Discord user ID

pels.

### Describe your question in detail.

I want to do some test when creating token with `0x4::token::create_numbered_token` and  `0x4::token::create` but I need the token_object after minting to do the test. when I use `0x4::token::create_named_token` I am able to generate the token_object by using 
```
let token_address = token::create_token_address(
            &signer::address_of(&creator),
            &collection_name,
            &token_name
        );
        object::address_to_object<Token>(token_address)
```
but anytime I tried to do that for the other method it give an error `error::not_found(EOBJECT_DOES_NOT_EXIST)`

### What error, if any, are you getting?

`error::not_found(EOBJECT_DOES_NOT_EXIST)`

### Which operating system are you using?

macOS

### Which SDK or tool are you using? (if any)

N/A

**Comment by MoonShiesty** [comment]
can you post the rest of your code?

**Reply by Pelumi527** [reply]
https://gist.github.com/Pelumi527/b3a00f9af4e9c7fbfe25aab17907a8a9
